diff --git a/drivers/amlogic/usb/dwc_otg/310/Makefile b/drivers/amlogic/usb/dwc_otg/310/Makefile
index 5e0e0ae..383e11c 100644
--- a/drivers/amlogic/usb/dwc_otg/310/Makefile
+++ b/drivers/amlogic/usb/dwc_otg/310/Makefile
@@ -7,7 +7,7 @@ DWC_OTG_NAME	= dwc_otg
 COMMON_PORT	= common_port
 BUS_INTERFACE = -DLM_INTERFACE
 
-KBUILD_CPPFLAGS	+= -DDEBUG 
+#KBUILD_CPPFLAGS	+= -DDEBUG 
 
 KBUILD_CPPFLAGS		+= -Dlinux -DDWC_HS_ELECT_TST
 #KBUILD_CPPFLAGS	+= -DDWC_EN_ISOC
@@ -30,4 +30,4 @@ $(DWC_OTG_NAME)-objs	+= dwc_otg_cfi.o
 endif
 $(DWC_OTG_NAME)-objs	+= $(COMMON_PORT)/dwc_common_linux.o
 
-obj-$(CONFIG_USB_DWC_OTG_HCD)	+= $(DWC_OTG_NAME).o
\ No newline at end of file
+obj-$(CONFIG_USB_DWC_OTG_HCD)	+= $(DWC_OTG_NAME).o
